lib/cli/args.js

```diff
@@ -382,6 +382,13 @@ const driverUpdateArgs = [
     example: 'xcuitest',
     help: 'Name of the driver to update, or the word "all" to update all. To see available updates, run "appium driver list --installed --updates"',
   }],
+  [['--force'], {
```

Comment:
the name of this arg does seem too generic to me

Reply:
it makes sense to have this option in the config

Reply:
what do you mean when you say have it in the config? you mean, if there's a yaml field in the config, then always run as if '--force' was used?